### DISCUSSION TITLE
병원 목록 데이터 없음 문제

### DISCUSSION THREAD

**Discussion by kimjaee** [changdoc/naver-vaccine-macro#228] [Q&A]
안녕하세요 개발자님!
병원 예약 신청 페이지 url 알아내는 방법을 하고있었는데

탐지할 지역의 graphql result을 입력해두어야 예약 신청 페이지로 이동 가능합니다 라는 첫번째 창이 뜨고
병원 목록 데이터가 없습니다. 현재 지도에서 새로고침을 수행한 graphql result를 매크로 페이지에 입력하여 병원 목록을 생성한 뒤 동작시켜주세요 라는 두번째 창이 뜹니다

개발자 도구를 켜고 새로고침을 해봐도  graphql result가 나오질 않아요ㅠㅠ

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/4
https://github.com/changdoc/naver-vaccine-macro/issues/11
두 내용  먼저 진행이 되어야하는데 잘 진행된 상태일까요?

**Reply by presso22** [reply]
#11에서 graphql result 아래 텍스트 상자에 붙여넣으라는데 대체 그 텍스트 상자 어디서 찾는건가요?? 어디서 뭐 눌러야 나오는거에요?

**Reply by changdoc** [reply]
https://github.com/changdoc/naver-vaccine-macro/issues/11 이쪽 가이드 보시면
![image](https://user-images.githubusercontent.com/1879494/129135038-2f710ebf-8e9a-40e7-a537-5bcd8695aa1c.png)
https://github.com/changdoc/naver-vaccine-macro/issues/11 매크로 창을 눌러 스크롤을 조금 내리시면 보일거에요.

**Reply by presso22** [reply]
감사합니다. 
![aaaaa22](https://user-images.githubusercontent.com/88812138/129138528-09e5cea7-e195-4f5d-8b5a-ce5d1a6e5731.png)
입력해서 넣었는데 "예약신청" 페이지에  "자동예약 시작" 누르면 원래 화면이 계속 새로고침 되었던거로 기억하는데 눌러도 "예약완료" 로 멈춰있네요. 이게 잘 되고있는건가요? 저는 자동예약 정지 버튼이 안나옵니다. 뭐가 문제일까요?

**Reply by presso22** [reply]
무슨 오류인가요?
